ui/app/components/inference/VariantResponseModal.tsx

```diff
@@ -67,13 +69,18 @@ export function VariantResponseModal({
     title,
     response,
     errorMessage,
+    children,
```

Comment:
This should have been flagged before but you shouldn't create new React components within React components. Let's extract this and pass additional props as needed.